python/doc/demos/NODDI_turorial_01.md

```diff
@@ -0,0 +1,124 @@
+# Tutorial for python-AMICO/NODDI
```

Comment:
Title could be "Fitting the NODDI model".
Using the words "AMICO" and "Python" are redundant as we are in the project AMICO-python.